index.html

```diff
@@ -94,32 +94,40 @@ <h2 class="header">Things we've done</h2>
             <div class="col-12 col-lg-8 images">
                 <div class="row">
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.jpg" alt="The logo for CSH Schedule Maker" />
-                        </picture>
+                        <a href="https://schedule.csh.rit.edu/">
+                            <picture>
+                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.webp" type="image/webp" />
+                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.jpg" type="image/jpeg" />
+                                <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.jpg" alt="The logo for CSH Schedule Maker" />
+                            </picture>
+                        </a>
                     </div>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" alt="Two custom arcade machines in the CSH library" />
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/Arcade-Controls">
```

Comment:
Would love some confirmation by Brandon Chiu/R&D/anyone w/ knowledge of the arcade cabinets if this is still in use, as the repo hasn't been touched in 10 years.